dae/dae/backends/impala/parquet_io.py

```diff
@@ -59,14 +59,51 @@ def __len__(self):
         return len(self.data['summary_variant_index'])
 
 
+class ContinuousParquetFileWriter(object):
+    """
+    Class that automatically writes to a given parquet file when supplied
+    enough data. Automatically dumps leftover data when closing into the file
+    """
+    def __init__(
+                self, filepath, schema,
+                filesystem=None, rows=10000):
+        self._data = ParquetData(schema)
+        schema = schema.to_arrow()
+        path = os.path.dirname(filepath)
+        if not os.path.exists(path):
+            os.makedirs(path)
+        self._writer = pq.ParquetWriter(
+                filepath,
+                schema,
+                compression='snappy',
+                filesystem=filesystem)
+        self.rows = rows
+
+    def _write_table(self):
+        print('WRITING')
+        self._writer.write_table(self._data.build_table())
+
+    def data_append(self, attr_name, value):
+        self._data.data_append(attr_name, value)
+
+    def write_if_overflowing(self):
```

Comment:
if the writer is 'automatic' probably we should put this check inside data_append